### PR TITLE
Java script functions

### PR DESCRIPTION
**Pull request by glecie18**
for dev review

### CODE REVIEW

File: Creating Function/index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <title>JavaScript Basics</title>
+</head>
+<body>
+<script src="script.js"></script>
```
Comment (chemagaton24) [inline Creating Function/index.html]: I'm getting error when running the project, Looks like scripts should be `script_CF.js`
image

```suggestion
<script src="script_CF.js"></script>
```

File: Uses of Closure/setUpHandlers.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+var buttons = document.getElementsByTagName('button');
```
Comment (chemagaton24) [inline Uses of Closure/setUpHandlers.js]: @glecie18 There is a problem with the console output when clicking buttons, There result only outputs `Third`

https://github.com/user-attachments/assets/19cc5647-15f3-45d4-a45b-4f3e496efcc1

I believe the output should correlate with the button text which should output `Second` if Second button is clicked same goes with other buttons.

Also, Please don't use `var` keyword when declaring variables as it's prone to bugs and it doesn't respect block of scope.

Please see my suggested fix, to solve the issue
```suggestion
const buttons = document.getElementsByTagName('button');
```

File: Uses of Closure/setUpHandlers.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+for(var i = 0; i < buttons.length; i += 1) {
+	var button = buttons[i];
```
Comment (chemagaton24) [inline Uses of Closure/setUpHandlers.js]:
```suggestion
	const button = buttons[i];
```

File: Uses of Closure/setUpHandlers.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+	var buttonName = button.innerHTML;
```
Comment (chemagaton24) [inline Uses of Closure/setUpHandlers.js]:
```suggestion
	const buttonName = button.innerHTML;
```

**Comment by glecie18** [comment]
I will PR latest one for the commit changes

**Pull request closed without merging**